### PR TITLE
- integrates Trapi 1.0

### PR DESCRIPTION
**Pull request by YaphetKG**
- makes get_kg method do temporary trapi 0.9 conversion to preserve UI

### CODE REVIEW

**Review by cschreep — DISMISSED** [review]

**Review by cschreep — APPROVED** [review]

File: src/dug/core/annotate.py [hunk 1]
```diff
@@ -148,11 +148,7 @@ def expand_identifier(self, identifier, query_factory, kg_filename):
                 data=query).json()
 
             # Case: Skip if empty KG
-            logger.debug(response)
-            if 'knowledge_graph' not in response:
-                logger.debug(f"Did not find a knowledge graph for {query}")
-                return []
-            if not len(response['knowledge_graph']['nodes']):
+            if not len(response["message"]['knowledge_graph']['nodes']):
```
Comment (cschreep) [inline src/dug/core/annotate.py:151]: Accessing nested dicts should have error handling in case one of the keys is missing

**Review by cschreep — CHANGES_REQUESTED** [review]

File: src/dug/core/annotate.py (hunk 1 above)
Comment (YaphetKG) [reply src/dug/core/annotate.py]: modified to 
```
            if not len(response.get("message", {}).get("knowledge_graph", {}).get("nodes", [])):
                logger.debug(f"Did not find a knowledge graph for {query}")
                logger.debug(f"{self.url} returned response: {response}")

Comment (cschreep) [reply src/dug/core/annotate.py]: 👍🏻  although maybe 
```
try:
    nodes = response["message"]['knowledge_graph']['nodes']
except KeyError:
    ....
```

Comment (YaphetKG) [reply src/dug/core/annotate.py]: updated it to 
```            
          try:
                if not len(response["message"]["knowledge_graph"]["nodes"]):
                    logger.debug(f"Did not find a knowledge graph for {query}")
                    logger.debug(f"{self.url} returned response: {response}")
                    return []
            except KeyError as e:
                logger.error(f"Could not find key: {e} in response: {response}")

Comment (cschreep) [reply src/dug/core/annotate.py]: 👍🏻  much cleaner

**Review by cschreep — APPROVED** [review]

**Pull request merged**